### PR TITLE
added ens token events

### PR DESCRIPTION
**Pull request by cagostino**
## What?
ie: Add support for uniswap contract events 

## How? 
ie: I used [abi-parser](https://nansen-contract-parser-prod.web.app/) or cli tools to build the table defenitions

## Related PRs (optional)
PRs that are related to this or may need to be deployed before this PR

### CODE REVIEW

**Comment by cagostino** [comment]
Used the Contract Parser: https://contract-parser.d5.ai

**Comment by cagostino** [comment]
Gotcha, I moved them now.

File: dags/resources/stages/parse/table_definitions/ens/ENSToken_event_Claim.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            ],
+            "name": "Claim",
+            "type": "event"
+        },
+        "contract_address": "0xc18360217d8f7ab5e7c516566761ea12ce7f9d72",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "",
```
Comment (medvedev1088) [inline dags/resources/stages/parse/table_definitions/ens/ENSToken_event_Claim.json:27]: Should be "dataset_name": "ens"

File: dags/resources/stages/parse/table_definitions/ens/ENSToken_event_DelegateChanged.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            ],
+            "name": "DelegateChanged",
+            "type": "event"
+        },
+        "contract_address": "0xc18360217d8f7ab5e7c516566761ea12ce7f9d72",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "",
```
Comment (araa47) [inline dags/resources/stages/parse/table_definitions/ens/ENSToken_event_DelegateChanged.json:33]: @cagostino can you update the dataset in all the files

File: dags/resources/stages/parse/table_definitions/ens/ENSToken_event_DelegateVotesChanged.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            ],
+            "name": "DelegateVotesChanged",
+            "type": "event"
+        },
+        "contract_address": "0xc18360217d8f7ab5e7c516566761ea12ce7f9d72",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "",
```
Comment (araa47) [inline dags/resources/stages/parse/table_definitions/ens/ENSToken_event_DelegateVotesChanged.json:33]: same as above

File: dags/resources/stages/parse/table_definitions/ens/ENSToken_event_MerkleRootChanged.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            ],
+            "name": "MerkleRootChanged",
+            "type": "event"
+        },
+        "contract_address": "0xc18360217d8f7ab5e7c516566761ea12ce7f9d72",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "",
```
Comment (araa47) [inline dags/resources/stages/parse/table_definitions/ens/ENSToken_event_MerkleRootChanged.json:21]: same as above

File: dags/resources/stages/parse/table_definitions/ens/ENSToken_event_OwnershipTransferred.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+            ],
+            "name": "OwnershipTransferred",
+            "type": "event"
+        },
+        "contract_address": "0xc18360217d8f7ab5e7c516566761ea12ce7f9d72",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "",
```
Comment (araa47) [inline dags/resources/stages/parse/table_definitions/ens/ENSToken_event_OwnershipTransferred.json:27]: same as above

File: dags/resources/stages/parse/table_definitions/ens/ENSToken_event_Transfer.json [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+            ],
+            "name": "Transfer",
+            "type": "event"
+        },
+        "contract_address": "0xc18360217d8f7ab5e7c516566761ea12ce7f9d72",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "",
```
Comment (araa47) [inline dags/resources/stages/parse/table_definitions/ens/ENSToken_event_Transfer.json:33]: same as above

**Review by araa47 — CHANGES_REQUESTED** [review]
please add the dataset name to all the json files

**Comment by araa47** [comment]
@cagostino , the github action will succeed if the files are formatted correctly :)

File: dags/resources/stages/parse/table_definitions/ens/ENSToken_event_DelegateChanged.json (hunk 2 above)
Comment (cagostino) [reply dags/resources/stages/parse/table_definitions/ens/ENSToken_event_DelegateChanged.json:33]: oops yes will do

**Pull request merged**